lib/param_parsing/environment.go

```diff
@@ -3,6 +3,11 @@ package param_parsing
 import "os"
 
 func GetParamsFromEnvironment(params Params) Params {
-	params.Version = os.Getenv("TF_VERSION")
+	if envVersion := os.Getenv("TF_VERSION"); envVersion != "" {
+		params.Version = envVersion
+	}
+	if envProduct := os.Getenv("TF_PRODUCT"); envProduct != "" {
```

Comment:
Should the option to use Env var be reflected in documentation?
Ref: https://tfswitch.warrensbox.com/usage/commandline/#use-environment-variables

Reply:
Done in 1af481d